src/modules/job-exec/exec_config.c

```diff
@@ -24,12 +24,13 @@
 #include "exec_config.h"
 #include "ccan/str/str.h"
 
-static const char *default_cwd = "/tmp";
-static const char *default_job_shell = NULL;
-static const char *flux_imp_path = NULL;
-static const char *exec_service = "rexec";
-static int exec_service_override = 0;
-static json_t *sdexec_properties = NULL;
+/* Globals initialized in config_init() */
+static const char *default_cwd;
+static const char *default_job_shell;
+static const char *flux_imp_path;
+static const char *exec_service;
+static int exec_service_override;
+static json_t *sdexec_properties;
```

Comment:
The `default_` prefixed variables are not set by configuration so they probably should not be set by `config_init()`.  They are just static defaults.

For the values that are configured, could we put them in a struct to avoid re-declaring the "tmp" versions in the function?